fdbserver/include/fdbserver/ProxyCommitData.actor.h

```diff
@@ -37,6 +35,8 @@
 #include "fdbserver/ResolverInterface.h"
 #include "flow/IRandom.h"
 
+#include <unordered_map>
```

Comment:
nit: can be removed?

Reply:
ah, how can I turn off the autocomplete in VSCode?